hugo/assets/scripts/domainmodel/domainmodel-tools.ts

```diff
@@ -22,6 +13,24 @@ export function getTreeNode(ast: AstNode): TreeNode {
         }
     }
 
+    // create a TreeNode from a PackageDeclaration
+    const getPackageDeclarationTreeNode = (p: PackageDeclaration): TreeNode => {
+        return {
+            ...p,
+            children: p.elements.map(e => getTreeNode(e))
+        }
+    }
+
+    // create a TreeNode from any DomainModelElement
+    const getTreeNode = (e: DomainModelElement): TreeNode => {
```

Comment:
You can remove the comment by giving proper names for `getTreeNode`. Just name it `getDomainModelTreeNode`. Same for the comments above. If the comment gives no extra information to the next line, it is basically useless and a danger of becoming outdated when the code changes again.
Comments are good for stuff that is not written in code (like motivations or hidden knowledge).